cirq-google/cirq_google/json_test_data/PhasedFSimCalibrationResult.json_inward

```diff
@@ -54,5 +54,8 @@
     "characterize_chi": false,
     "characterize_gamma": false,
     "characterize_phi": true
-  }
+  },
```

Comment:
the `_inward` files should omit the new fields to test backwards compatibility

Reply:
Done